.editorconfig

```
@@ -0,0 +1,3 @@
+[*.sh]
```

As I know, the `.editorconfig` on a repo root should be marked as root.

```suggestion
root = true

[*.sh]
```